README.md

```diff
@@ -1,5 +1,6 @@
 # Go Repository Template
 
+[![Keep a Changelog](https://img.shields.io/badge/changelog-Keep%20a%20Changelog-%23E05735)](CHANGELOG.md)
```

Comment:
I think badges are nice, when only a few of them are used and they point to a really important stuff. When the project has more than 5, I believe most of the people ignore them. A very nice number of badges is 3, in my opinion 🙂 

![badges](https://user-images.githubusercontent.com/10604335/116686350-fa638b80-a9b3-11eb-98ab-19fcbdfe79c1.png)

For example, when I see these, my brain ignores all the green ones on the right. The `reference` badge is also ignored.

I remember:
* Keep a Changelog ← because it is the first badge
* v.0.14.0 and v1.16 ← I'm looking for a version which I should use
* 50% ← because it is red

What I think is really important for the user:
* license ← can I use it or not
* version ← I want quickly get the latest and use it
* docs (including tutorial, references and changelog) ← an easy way to navigate to it

For all other, I'm not interested. Of course this is my personal feeling 🙂

Reply:
Yeah, opinions 😄  I encourage to update the `README.md` after using this template so one can keep the badges he prefers.

Reply:
Okay, no problem 🙂 Where is the `Reopen` button? 😛